src/credentials/providers/sharedCredentialsProvider.ts

```diff
@@ -247,4 +247,13 @@ export class SharedCredentialsProvider implements CredentialsProvider {
     public static getCredentialsType(): string {
         return SharedCredentialsProvider.CREDENTIALS_TYPE
     }
+
+    public isSsoProfile(): boolean {
+        for (const propertyName of SSO_PROFILE_PROPERTIES) {
+            if (hasProfileProperty(this.profile, propertyName)) {
+                return true
```

Comment:
This will return true on the first matching property name. Is that expected, or should this validate all properties in `SSO_PROFILE_PROPERTIES`? `validateSsoProfile` appears to do similar work (it validates all properties though)...should that be used instead (i.e. everything's good if it returns `undefined`)?

Reply:
Good question, though could save that for a later PR.

Reply:
Resolved offline, this puts the credential through the pipeline which eventually runs validate.